.github/workflows/version-or-publish.yml

```diff
@@ -44,11 +44,9 @@ jobs:
             git commit -am "Format"
             git push
           fi
-        env:
-          GITHUB_TOKEN: ${{ secrets.TYPESCRIPT_BOT_TOKEN }}
       - uses: changesets/action@v1
         with:
-          publish: pnpm ci:publish
```

Comment:
We should delete this package.json script if we're not going to use it

Reply:
Oops, I thought I committed that